src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,124 @@
 package mate.jdbc;
 
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.CarService;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        ManufacturerService manufacturerService
+                = (ManufacturerService) injector.getInstance(ManufacturerService.class);
+        System.out.println("Manufacturer Service testing:");
+        Map<String, Manufacturer> manufacturersMap
+                = runExampleOfUsageManufacturerService(manufacturerService);
+
+        DriverService driverService = (DriverService) injector.getInstance(DriverService.class);
+        System.out.println("\nDriver Service testing:");
+        Map<String, Driver> driversMap = runExampleOfUsageDriverService(driverService);
+
+        CarService carService = (CarService) injector.getInstance(CarService.class);
+        System.out.println("\nCar Service testing:");
+        runExampleOfUsageCarService(carService, manufacturersMap, driversMap);
+    }
+
+    private static Map<String, Manufacturer> runExampleOfUsageManufacturerService(
+            ManufacturerService manufacturerService) {
+        System.out.println("Create:");
+        Manufacturer opelManufacturer = new Manufacturer(null, "Opel", "Germany");
+        Manufacturer fiatManufacturer = new Manufacturer(null, "Fiat", "France");
+        Manufacturer renaultManufacturer = new Manufacturer(null, "Renault", "France");
+        System.out.println(manufacturerService.create(opelManufacturer));
+        System.out.println(manufacturerService.create(fiatManufacturer));
+        System.out.println(manufacturerService.create(renaultManufacturer));
+        Map<String, Manufacturer> manufacturersMap = new HashMap<>();
+        manufacturersMap.put(opelManufacturer.getName(), opelManufacturer);
+        manufacturersMap.put(fiatManufacturer.getName(), fiatManufacturer);
+        manufacturersMap.put(renaultManufacturer.getName(), renaultManufacturer);
+
+        System.out.println("Delete:");
+        System.out.println(manufacturerService.delete(fiatManufacturer.getId()));
+        manufacturersMap.remove(fiatManufacturer.getName());
+
+        System.out.println("Get all:");
+        List<Manufacturer> manufacturers = manufacturerService.getAll();
+        manufacturers.forEach(System.out::println);
+        return manufacturersMap;
+    }
+
+    private static Map<String, Driver> runExampleOfUsageDriverService(DriverService driverService) {
```

Comment:
it's good idea to use a separate method, but maybe you should use simpler name

Reply:
ok)

Reply:
agree with @AnastasiiaVariuchuk let's make your methods names more concise and clear